### PR TITLE
Update the webpage

### PR DESCRIPTION
**Pull request by bzsolt**

### CODE REVIEW

File: 02.API-REFERENCE.md [hunk 1]
```diff
@@ -660,6 +729,8 @@ Possible values:
  - JERRY_TYPEDARRAY_INT32 - represents the Int32Array TypedArray
  - JERRY_TYPEDARRAY_FLOAT32 - represents the Float32Array TypedArray
  - JERRY_TYPEDARRAY_FLOAT64 - represents the Float64Array TypedArray
+ - JERRY_TYPEDARRAY_BIGINT64 - represents the BigInt64Array TypedArray
+ - JERRY_TYPEDARRAY_BIGUINT64 - represents the BigUint64Array TypedArray
```
Comment (ossy-szeged) [inline 02.API-REFERENCE.md:733]: Please add *Changed in version [[NEXT_RELEASE]]*: Added `JERRY_TYPEDARRAY_BIGINT64`, `JERRY_TYPEDARRAY_BIGUINT64` values.

Comment (bzsolt) [reply 02.API-REFERENCE.md:733]: Ok, I am going to add it to the doc first, then update this PR.

Comment (bzsolt) [reply 02.API-REFERENCE.md:735]: I've added the suggested lines as https://github.com/jerryscript-project/jerryscript/pull/4259 landed.

**Review by rerobika — APPROVED** [review]
LGTM

**Pull request merged**